applications/tari_app_grpc/src/conversions/block_header.rs

```diff
@@ -81,11 +81,11 @@ impl TryFrom<grpc::BlockHeader> for BlockHeader {
             height: header.height,
             prev_hash: header.prev_hash,
             timestamp,
-            input_mr: header.input_mr,
-            output_mr: header.output_mr,
-            witness_mr: header.witness_mr,
+            input_mr: FixedHash::try_from(header.input_mr).expect("Array size 32 cannot fail"),
```

Comment:
This can fail. The error should be propagated 
```suggestion
            input_mr: FixedHash::try_from(header.input_mr).ok_or("Input_MR was not 32 bytes".to_string())?,
```